src/core/reopt_inputs.jl

```diff
@@ -738,17 +742,19 @@ end
 
 """
     function setup_boiler_inputs(s::AbstractScenario, max_sizes, min_sizes, existing_sizes, cap_cost_slope, boiler_efficiency,
-        om_cost_per_kw, production_factor, fuel_cost_per_kwh, heating_cf)
+        tech_renewable, om_cost_per_kw, production_factor, fuel_cost_per_kwh, heating_cf)
 
 Update tech-indexed data arrays necessary to build the JuMP model with the values for (new) boiler.
 This version of this function, used in BAUInputs(), doesn't update renewable energy and emissions arrays.
 """
 function setup_boiler_inputs(s::AbstractScenario, max_sizes, min_sizes, existing_sizes, cap_cost_slope, boiler_efficiency,
-                            om_cost_per_kw, production_factor, fuel_cost_per_kwh, heating_cf)
+        tech_renewable_energy_fraction, om_cost_per_kw, production_factor, fuel_cost_per_kwh, heating_cf)
     max_sizes["Boiler"] = s.boiler.max_kw
     min_sizes["Boiler"] = s.boiler.min_kw
     existing_sizes["Boiler"] = 0.0
     boiler_efficiency["Boiler"] = s.boiler.efficiency
+    production_factor["Boiler", :] = ones(8760*s.settings.time_steps_per_hour)
```

Comment:
@zolanaj why is this needed if `production_factor["Boiler", :] = get_production_factor(s.boiler)` is down below?

Reply:
> @zolanaj why is this needed if `production_factor["Boiler", :] = get_production_factor(s.boiler)` is down below?

it isn't needed!  not sure how I put that in there but I removed it in https://github.com/NREL/REopt.jl/commit/219096f7ad6492e1c389effa68cb278726fe6c69